### DISCUSSION TITLE
rdkit.Avalon.pyAvalonTools module

### DISCUSSION THREAD

**Discussion by urekam712** [rdkit/rdkit#6847] [Development]
Need clean and clear cut explain and usage on this module with valid examples to explore it

**Comment by bp-kelley** [comment]
pyAvalonTools is a historical wrapper to Bernd Rohde‘s cheminformatics tools.

While it has some nice functionality, I don’t expect it to ever have full documentation.

**Comment by bp-kelley** [comment]
The current documentation is a good example of what it was used for, all of these have been superseded with rdkit functionality.

https://www.rdkit.org/docs/source/rdkit.Avalon.pyAvalonTools.html
